roles/callhome/node/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+scale_install_callhome_repository_url: http://9.11.84.162/gpfs_rpms/
+
+scale_callhome_rpms:
+  - gpfs.callhome-ecc-client
+  - gpfs.java
+scale_rpmversion: "{{ scale_version | regex_replace('^([0-9.]+)\\.([0-9])$', '\\1-\\2') }}"
+scale_install_localpkg_tmpdir_path: /tmp
+
+extracted_default_path: "/usr/lpp/mmfs"
```

Comment:
We probably need a better way to re-use the definitions here, we are duplicating these variables when we should all point to the same one in all our roles.

Reply:
@whowutwut Today i was discussing with the team that we will move this  in the common role . but currently i was not trying to touch existing role that's the reason we added here.